specification/patient-care-aggregator-api-record-service.yaml

```diff
@@ -48,15 +48,22 @@ info:
 
     To see how to include the API key with your API request, see the endpoint description below.
 
+    Each Portal will be provided with a different Client ID to be used for each Resource type they support, e.g.:
+
+    <portal>-appointments
+    <portal>-documents
+    <portal>-questionnaires
+    
+    The same API bearer token will used across all Client IDs.
+
     ## Environments and testing 
 
-    | Environment       | Base URL                                                               |
-    | ----------------- | ---------------------------------------------------------------------- |
-    | Sandbox           | `https://tbc`        |
-    | Integration test  | `https://tbc`        |
-    | Performance test  | `https://tbc`        |
-    | End-to-end test   | `https://tbc`        |
-    | Production        | `https://tbc`        |
+    | Environment                         | Base URL                                                               |
+    | ------------------------------------| ---------------------------------------------------------------------- |
+    | Connectivity and data quality tests | `https://integration.servita-demo.co.uk`                               |
```

Comment:
We have referred to this as "sandbox" testing elsewhere so can we use that consistent language?

Reply:
Changed